test/integration/server-test.ts

```diff
@@ -6,6 +6,18 @@ import { promisify } from "util";
 import { Webhooks } from "../../src";
 import pushEventPayload from "../fixtures/push-payload.json";
 
+const signatureHeaders = [
+  {
+    algorithm: "sha1",
+    signature: "sha1=f4d795e69b5d03c139cc6ea991ad3e5762d13e2f",
+  },
+  {
+    algorithm: "sha256",
+    signature:
+      "sha256=4864d2759938a15468b5df9ade20bf161da9b4f737ea61794142f3484236bda3",
```

Comment:
WIP. After adding coverage for `sha256` some tests failing. Not sure if the sha256 is incorrect or there's a bug at sign/verify stage. Tomorrow I will keep checking